packages/core-utils/README.md

```diff
@@ -1,5 +1,9 @@
 Includes various utility functions to assist with common operations dealing with OpenTripPlanner data.
 
+This package depends on [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), which is not supported by older browsers or node versions. [As can be seen here](https://caniuse.com/?search=Intl.DateTimeFormat), support is fairly significant across major browsers. Should older browsers need to be targeted, a polyfill can be added. Such a polyfill is available [here](https://formatjs.io/docs/polyfills/intl-datetimeformat/).
+
```

Comment:
TIL about [the Year 2038 problem](https://en.wikipedia.org/wiki/Year_2038_problem) at that formatjs website.

Reply:
All the nuclear power plants running Windows 2000...